aten/src/ATen/native/native_functions.yaml

```diff
@@ -270,6 +270,8 @@
   dispatch:
     CPU, CUDA: angle_out
 
+- func: attn(Tensor q, Tensor k, Tensor v) -> (Tensor, Tensor)
```

Comment:
nit: you can give names to the outputs so that the python binding will return a NamedTuple instead of just a tuple.

Reply:
Just added that to a previous commit.